openedx/tests/xblock_integration/test_review_xblock.py

```diff
@@ -363,12 +363,13 @@ def test_review_problems(self, num_desired):
                 'section': review_section_actual.location.name,
             }
         ))
+        print response
```

Comment:
@Dillon-Dumesnil not sure you meant to leave this in there?

Reply:
... There's always something. One sec